modules/checklists/checklists.php

```diff
@@ -365,6 +365,8 @@ public function init()
             do_action('publishpress_checklists_load_requirements');
 
             add_filter('publishpress_checklists_rules_list', [$this, 'filterRulesList']);
+
+            add_filter('publishpress_checklists_requirement_list', [$this, 'filterRequirementsRule'], 1000, 3);
```

Comment:
@olatechpro  Just a minor issue. It says 3 params, but the function accepts only 2

Reply:
But as I can see the function really uses only one. I think we can remove this param, and the `$post` param of method as well, since it is not used in the method body.

Reply:
Updated. I leave the 1000 alone so the function can run last in case any plugin hook to same filter